UnitySDK/Assets/ML-Agents/Examples/Bouncer/Scripts/BouncerAgent.cs

```diff
@@ -1,128 +1,124 @@
-﻿using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using MLAgents;
 
-public class BouncerAgent : Agent {
-
+public class BouncerAgent : Agent
+{
     [Header("Bouncer Specific")]
     public GameObject banana;
     public GameObject bodyObject;
-    Rigidbody rb;
-    Vector3 lookDir;
+    Rigidbody m_Rb;
+    Vector3 m_LookDir;
     public float strength = 10f;
-    float jumpCooldown;
+    float m_JumpCooldown;
     int numberJumps = 20;
-    int jumpLeft = 20;
+    int m_JumpLeft = 20;
```

Comment:
Any idea why this went `m_JumpLeft` but it didn't change `numberJumps`?

Reply:
`numberJumps` should be const and renamed `k_NumberJumps`